Update account create docs (#2517)
foundry-rs/starknet-foundry: docs/src/starknet/account.md

- create account with the `sncast account create` command

```shell
$ sncast \
    account create \
    --url http://127.0.0.1:5050 \
    --name some-name
```

```
command: account create
add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
address: 0x34ae54182d04754d8043189afd315a808d4bea1a63862b3b05aa78b37756d7b
max_fee: 180527346330500
message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens or an equivalent amount of ETH tokens. It is good to send more in the case of higher demand.

To see account creation details, visit:

> account: https://starkscan.co/search/34ae54182d04754d8043189afd315a808d4bea1a63862b3b05aa78b37756d7b

Reviewer: ```suggestion
account: https://starkscan.co/search/contract/34ae54182d04754d8043189afd315a808d4bea1a63862b3b05aa78b37756d7b
```

 >💡 **Info**
> When deploying on a Sepolia test network, you can also fund your account with artificial tokens via the [Starknet Faucet](https://starknet-faucet.vercel.app)

> >![image](images/starknet-faucet-sepolia.png)

Reviewer 2: ```
To deploy an account in the next step, you need to prefund it with either STRK or ETH tokens
```

This could say 

```
To deploy an account in the next step, you need to prefund it with either STRK or an equivalent amount of ETH token
```

transaction_hash: 0x20b20896ce63371ef015d66b4dd89bf18c5510a840b4a85a43a983caa6e2579

To see invocation details, visit:
transaction: https://starkscan.co/search/0x20b20896ce...
```

For a detailed CLI description, see [account deploy command reference](../appendix/sncast/account/deploy.md).

> 💡 **Info**
> You can also choose to pay in Ether by setting `--fee-token` to `eth`.

> ## Additional features

Reviewer 2: Maybe let's rename it `Managing Accounts`.

### [`account list`](../appendix/sncast/account/list.md)
List all accounts saved in `accounts file`, grouped based on the networks they are defined on.

```shell
$ sncast account list
```

```shell

> $ sncast --accounts-file my-account-file.json account list

Reviewer 2: I'd group output with the `sncast account list` and just mention below that you can also lists accounts in specific file by providing `--accounts-file`. But I wouldn't show the whole output for that.

transaction_hash: 0x20b20896ce63371ef015d66b4dd89bf18c5510a840b4a85a43a983caa6e2579

To see invocation details, visit:
transaction: https://starkscan.co/search/0x20b20896ce...
```

For a detailed CLI description, see [account deploy command reference](../appendix/sncast/account/deploy.md).

> 💡 **Info**
> You can also choose to pay in Ether by setting `--fee-token` to `eth`.

## Managing Accounts

> If you created an account with `sncast account create` it by default it will be saved in `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` file named as default file in below sections.

Reviewer 2: ```suggestion
If you created an account with `sncast account create` it by default it will be saved in `~/.starknet_accounts/starknet_open_zeppelin_accounts.json` file which we call `default accounts file` in the following sections.
```

```shell
$ sncast \
    account create \

>     --url http://127.0.0.1:5050 \

Reviewer 2: I'd move `### ``account create`` With Salt Argument` to advanced use cases as well.

[...]
```

### [`account delete`](../appendix/sncast/account/delete.md)

Delete an account from `accounts-file` and its associated Scarb profile. If you pass this command, you will be asked to confirm the deletion.

```shell
$ sncast \

>     --accounts-file my-account-file.json \

Reviewer 2: Don't think this argument is necessary be default?

    account delete \
    --name some-name \
    --network alpha-sepolia
```

### [`account add`](../appendix/sncast/account/add.md)

> To import an account into the file holding the accounts info (`~/.starknet_accounts/starknet_open_zeppelin_accounts.json` by default), use the `account add` command.

Reviewer 2: Maybe this should just be a separate section somewhere on this page similar to this

```md
## Accounts File

By default the file is this ... but you can override it with `--accounts-file` 

maybe a single example there.
```

I think this could maybe be in advanced usecases.

Reviewer 2: Actually we already have that below, so maybe consider extending it and don't add these explanations anywhere else.

Author: I think that this paragraph fits well in this place, and creates a full composition of managing account 
```md
## Managing Accounts
### account add
### account list
### account delete
```
I've updated the docs, wdyt?

Reviewer 2: We can keep it, please add an issue to backlog to link to new docs that will be created as part of #2565

$ sncast \
    account create \
    --name some-name \
    --url http://127.0.0.1:5050 \
    --class-hash 0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6
```

#### [`account create`](../appendix/sncast/account/create.md) With Salt Argument

> Salt will not be randomly generated if it's specified with `--salt`.

Reviewer 2: ```suggestion
Instead of random generation, salt can be specified with `--salt`.
```

```shell
$ sncast \
    account create \
    --url http://127.0.0.1:5050 \
    --name some-name \
    --salt 0x1
```

#### Additional features provided with `account add/create`

##### Specifying [`--accounts-file`](../appendix/sncast/account/create.md#create)

Account information such as `private_key`, `class_hash`, `address` etc. will be saved to the file specified by `--accounts-file` argument, 

> if not provided, the default file will be used.

Reviewer 2: ```suggestion
if not provided, the `default accounts file` will be used.
```
